### ISSUE TITLE
Please release a version to PyPI

### ISSUE THREAD

**Issue by hggh** [graphite-project/ceres#31]
would be nice if we can release a version to pypi.

**Comment by SEJeff** [comment]
Give us a week or so to look through the existing backlog of tickets and merge any outstanding PRs, but this should be doable soonish.

**Comment by hggh** [comment]
:+1:

**Comment by mleinart** [comment]
I dont see a problem uploading a version of some kind, but I'll note that graphite components have always been released lockstep with everything on the same version. Given that the master branch of carbon still doesnt have ceres support merged in, that isn't possible in this case.

Perhaps we just name this 0.10.dev1 or 0.10.a1 - that looks to be what [PEP-0440](https://www.python.org/dev/peps/pep-0440/#pre-releases) wants.

**Comment by krak3n** [comment]
Didn't get round to it then :stuck_out_tongue:

**Comment by deniszh** [comment]
I'm wondering if ceres compatible with master carbon... Maybe somebody knows. @iain-buclaw-sociomantic ?

**Comment by deniszh** [comment]
I created issue #45 for discussing Ceres future.

**Comment by iain-buclaw-sociomantic** [comment]
Ceres is not only compatible with master carbon, it runs using half the CPU!

I think megacarbon should be considered dead, as the various PRs I raised last year pretty much hammered the final nail in its coffin - graphite-project/carbon#484,  graphite-project/carbon#508, and graphite-project/carbon#596.

**Comment by deniszh** [comment]
That's great, @iain-buclaw-sociomantic !
Then we still have a chance to include it to Graphite properly. :)

**Comment by iain-buclaw-sociomantic** [comment]
Yep.  Although I use graphite-api here, the last test I did using graphite-web appeared to be successful with a cursory test (https://github.com/graphite-project/graphite-web/pull/1719#issuecomment-274126421).  I think there are some known problems with brace matching in the ceres reader however, I'm not sure though, so don't hold me to that. :-)
